in_toto/runlib.py

```diff
@@ -398,7 +403,7 @@ def in_toto_run(name, material_list, product_list, link_cmd_args,
 
   if material_list:
     log.info("Recording materials '{}'...".format(", ".join(material_list)))
-  materials_dict = record_artifacts_as_dict(material_list)
+  materials_dict = record_artifacts_as_dict(material_list, follow_links=True)
 
```

Comment:
So `follow_links` is always set to true for someone using/calling `in_toto_run`?  The docstring for `in_toto_run` should probably mention this default behavior.

Reply:
Will do